docs/docs/getting-started.md

```diff
@@ -37,44 +36,45 @@ For convenience, we also provide a script `./run-docker.sh`, which combines all
 
```

Comment:
Replace the info shortcut copy with:

'For convenience, we provide a script `./run-docker.sh`, which combines all the following steps. It also performs a few sanity checks to prevent potential issues.